swan-lake/learn-the-language/distinctive-language-features/concurrency.md

```diff
@@ -506,26 +506,20 @@ In  Ballerina, you have a ``readonly`` type that represents immutable values. Th
 // The type of `s` is an immutable `string`` array.
 // The value is also constructed as an immutable value.
 readonly & string[] s = ["foo", "bar"];
- 
-type Row record {
-    // Both the field and its value are immutable.
-    readonly string[] k;
 
-    int v;
-};
+isolated function foo(string[] s) {
+    
+}
 
-table<Row> key(k) t = table [
-    // Can safely use `s` as a key.
-    {k: s, v: 17}
```

Comment:
Instead of removing this, can't we use a const map and make it work?

Reply:
I would keep the existing example as is and remove only 

```ballerina
table<Row> key(k) t = table [
    // Can safely use `s` as a key.
    {k: s, v: 17}
];
```

Reply:
Because we haven't described consts in this section.

Reply:
Changed